.github/workflows/ci.yml

```diff
@@ -195,10 +195,15 @@ jobs:
       ### Build the installer (if enabled)
 
       - name: Install utilities to build installer
-        if: ${{ matrix.installer }}
+        if: ${{ matrix.installer && startsWith(matrix.os, 'windows') }}
         run: |
           python -m pip install pyinstaller==5.13.2
 
+      - name: Install utilities to build installer
+        if: ${{ matrix.installer && !startsWith(matrix.os, 'windows') }}
+        run: |
+          python -m pip install pyinstaller==6.3.0
```

Comment:
Remove previous block, which installs an older version of pyinstaller

Reply:
Pyinstaller v6 isn't working for our Windows build, yet. The previous block is to allow Windows to use pyinstaller v5.x, conditional included.